### PR TITLE
Robins first commit

### PR DESCRIPTION
**Pull request by Robin-Ahlm**
Added my menu

### CODE REVIEW

File: menyer/for_goodness_shake/style.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    background-color: wheat;
+    text-align: center;
+    font-family: 'Architects Daughter', cursive;
+    max-width: 720px;
+    margin: auto;
+}
+
+.container {
+    position: relative;
+    display: flex;
```
Comment (frozenbanana) [inline menyer/for_goodness_shake/style.css]: The assignment specifies to not use flex or grid.

File: menyer/for_goodness_shake/style.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+.hamburgare {
+    position: absolute;
+    left: 240px;
+}
+
+.tillbehor {
+    position: absolute;
+    left: 480px;
+}
+
```
Comment (frozenbanana) [inline menyer/for_goodness_shake/style.css]: Avoid language mixing while coding. I recommend sticking with English always.

File: menyer/for_goodness_shake/index.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                <p class="menuItems">
+                    Extra pommes <br>
+                    Extra kött <br>
+                    Sallad <br>
+                </p>
+                <p class="itemPrices tillbehorPriser">
+                    15 kr <br>
+                    35 kr <br>
+                    25 kr <br>
+                </p>
```
Comment (frozenbanana) [inline menyer/for_goodness_shake/index.html]: Works but a bit unconventional. Normally a `table` could be used here.

**Review by frozenbanana — CHANGES_REQUESTED** [review]
A good start but need to be solved without flexbox nor Grid. 

Try putting the content in a table.

**Review by isakgranqvist2021 — APPROVED** [review]

File: menyer/for_goodness_shake/style.css (hunk 1 above)
Comment (Robin-Ahlm) [reply menyer/for_goodness_shake/style.css]: It no longer has flex or grid.

File: menyer/for_goodness_shake/style.css (hunk 2 above)
Comment (Robin-Ahlm) [reply menyer/for_goodness_shake/style.css]: This is fixed, don't know why I was using swedish for class names.

File: menyer/for_goodness_shake/index.html (hunk 3 above)
Comment (Robin-Ahlm) [reply menyer/for_goodness_shake/index.html]: I've removed what I had done and put it in all in a table instead.

**Review by frozenbanana — APPROVED** [review]
Nice job!

**Pull request merged**